index.html

```diff
@@ -0,0 +1,309 @@
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-5">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg" href="#">See Project</a>
+            </div>
+
+            <div class="art-printing bg-6">
+                <div class="art-printing-inner">
+                    <h3 class="art-printing-heading">Profesional Art Printing Data</h3>
+                    <p class="art-printing-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+                    </p>
+                    <ul class="art-printing-list">
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="btn-lg btn-dark-yellow" href="#">See Project</a>
+            </div>
+        </div>
+    </section>
+
+    <section class="about-section padding-left-right">
+        <div class="about-box">
+            <div class="about-text-box">
+                <h2 class="about-heading">About me</h2>
+                <p class="about-description">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a class="btn-sm" href="#">Get my Resume</a>
+            </div>
+
+            <div class="about-img"></div>
+        </div>
+
+        <hr class="about" />
+
+        <div class="about-categories">
+            <div class="categories">
+                <div class="categories-heading-box">
+                    <img src="img/language-icon.svg" alt="diamond icon" />
+                    <h3 class="categories-heading">Language</h3>
+                </div>
+                <ul class="categories-list">
+                    <li>Javascript</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+            </div>
+
+            <div class="categories">
+                <div class="categories-heading-box">
+                    <img src="img/framework-icon.svg" alt="rectangle icon" />
```

Comment:
- [ ] The images for each of the categories are also currently broken so kindly fix that as well :wink: 
- Your Design
![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/5089db2a-69d6-4dab-b396-308a8af011ac)

- The Figma Design
![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/1a962a24-57a2-43ee-af81-d8aa6f0fda03)